js/src/components/google-ads-account-card/claim-account-button.js

```diff
@@ -27,10 +27,15 @@ import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
  *
  * @param {Object} props React props.
  * @param {Function} [props.onClick] Function called back when the button is clicked.
+ * @param {boolean} [props.loading] Whether the button is in a loading state.
  * @param {Object} props.restProps Props to be forwarded to AppButton.
  */
-const ClaimAccountButton = ( { onClick = noop, ...restProps } ) => {
-	const { inviteLink } = useGoogleAdsAccountStatus();
+const ClaimAccountButton = ( {
+	onClick = noop,
+	loading = false,
```

Comment:
The original `restProps` also forwards the `loading` to `AppButton`. Destructuring and passing it may not be necessary.